src/eredis_sentinel.erl

```diff
@@ -113,10 +113,12 @@ handle_cast(_Request, State = #eredis_sentinel_state{}) ->
              {noreply, NewState :: #eredis_sentinel_state{}, timeout() | hibernate} |
              {stop, Reason :: term(), NewState :: #eredis_sentinel_state{}}).
 %% Current sentinel connection broken
+handle_info({'EXIT', _Pid, {connection_error, _}}, S) ->
+    {noreply, S};
 handle_info({'EXIT', Pid, _Reason}, #eredis_sentinel_state{conn_pid = Pid} = S) ->
     {noreply, S#eredis_sentinel_state{conn_pid = undefined}};
-
-handle_info({'EXIT', _Pid, _Reason}, S) ->
+handle_info({'EXIT', Pid, Reason}, S) ->
+    ?LOG_ERROR("eredis_sentinel: Exit from pid ~p with reason ~p and state ~p", [Pid, Reason, S]),
     {stop, normal, S};
```

Comment:
Is this always an error?

When eredis is started with sentinel options and later eredis is stopped with `eredis:stop(Pid)`, this message is logged? It's not an error. Also I don't think we should log the state. It looks like debugging.

How about instead exit with the same reason as the linked process? Then an error will be logged automatically if Reason is anything other than normal.

```suggestion
handle_info({'EXIT', _Pid, Reason}, S) ->
    {stop, Reason, S};
```